Source/Scene/PointCloud3DTileContent.js

```diff
@@ -781,8 +785,11 @@ define([
         var hasNormals = content._hasNormals;
         var hasBatchIds = content._hasBatchIds;
         var backFaceCulling = content._backFaceCulling;
+        var normalShading = content._normalShading;
         var vertexArray = content._drawCommand.vertexArray;
 
+        var normalsEnabled = hasNormals && normalShading;
```

Comment:
`backFaceCulling` will have issues because all the normal attribute setup is still behind the `normalsEnabled` boolean. Ultimately `normalsEnabled` should probably be removed.